app/main.js

```diff
@@ -686,6 +688,7 @@ function getTrayMenu () {
   } else if (!doNotDisturb) {
     trayMenu.push({
       label: i18next.t('main.pause'),
+      visible: !settings.get('hardcoreMode'),
```

Comment:
I think we should hide this options only when in break? 

Usecase for hardcore mode for me seems to be "make it even harder not to finish break early". I think I would still want to pause breaks when I to present something, or reset.

Reply:
In my opinion, real hardcore mode makes it impossible to avoid breaks.
I use the full screen layout, so hiding the pause/reset breaks menu items doesn't make sense for me.
I can pause/reset anytime then, so it's not a real hardcore mode IMO.